lib/ui/semantics.dart

```diff
@@ -366,6 +382,39 @@ class SemanticsFlag {
   /// used to implement accessibility scrolling on iOS.
   static const SemanticsFlag isHidden = const SemanticsFlag._(_kIsHiddenIndex);
 
+  /// Whether the semantics node represents an image.
+  ///
+  /// Platforms have special behavior for images.  TalkBack will inform the user
+  /// the labeled node is an image. iOS may use the image flag to avoid
+  /// inverting their color when using smart invert.
+  static const SemanticsFlag isImage = const SemanticsFlag._(_kIsImageIndex);
+
+  /// Whether the semantics node is a live region.
+  /// 
+  /// A live region indicates that updates to semantics node are important.
+  /// Platforms may use this information to make polite updates to the user to
+  /// inform them of this.
+  /// 
+  /// On Android, TalkBack will make a polite announcement of the first and
+  /// subsequent updates to the label of this node. This flag is not currently
```

Comment:
The iOS a11y bridge does something with this flag. What is not supported about it?